idb/postgres/internal/migrations/local_ledger/migration_test.go

```diff
@@ -0,0 +1,130 @@
+package localledger
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"github.com/algorand/go-algorand-sdk/client/v2/algod"
+	"github.com/algorand/go-algorand-sdk/encoding/json"
+	"github.com/algorand/go-algorand-sdk/encoding/msgpack"
+	algodConfig "github.com/algorand/go-algorand/config"
+	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/ledger"
+	"github.com/algorand/go-algorand/logging"
+	"github.com/algorand/go-algorand/rpcs"
+	"github.com/algorand/indexer/idb"
+	"github.com/algorand/indexer/util"
+	"github.com/algorand/indexer/util/test"
+	"github.com/jarcoal/httpmock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRunMigration(t *testing.T) {
+	httpmock.Activate()
+	defer httpmock.DeactivateAndReset()
+	// /genesis
+	httpmock.RegisterResponder("GET", "http://localhost/genesis",
+		httpmock.NewStringResponder(200, string(json.Encode(test.MakeGenesis()))))
+	// /v2/blocks/0 endpoint
+	blockCert := rpcs.EncodedBlockCert{
+		Block: test.MakeGenesisBlock(),
+	}
+	// /v2/blocks/0
+	httpmock.RegisterResponder("GET", "http://localhost/v2/blocks/0?format=msgpack",
+		httpmock.NewStringResponder(200, string(msgpack.Encode(blockCert))))
+
+	// /v2/blocks/1
```

Comment:
It looks like this could be a loop. The main benefit to this is that it becomes obvious that identical blocks are used
```
txn := test.MakePaymentTxn(0, 100, 0, 1, 1,
	0, test.AccountA, test.AccountA, basics.Address{}, basics.Address{})
for i := 1; i < 7; i++ {
	block, err := test.MakeBlockForTxns(test.MakeGenesisBlock().BlockHeader, &txn)
	assert.Nil(t, err)
	blockCert := rpcs.EncodedBlockCert{
		Block: block,
	}
	url := fmt.Sprintf("http://localhost/v2/blocks/%d?format=msgpack", i)
	httpmock.RegisterResponder("GET", url,
		httpmock.NewStringResponder(200, string(msgpack.Encode(blockCert))))
}
```

Maybe a regex would work also? Return the same thing until the blocks are in double digits:
```
	txn := test.MakePaymentTxn(0, 100, 0, 1, 1,
		0, test.AccountA, test.AccountA, basics.Address{}, basics.Address{})
	block, err := test.MakeBlockForTxns(test.MakeGenesisBlock().BlockHeader, &txn)
	assert.Nil(t, err)
	blockCert := rpcs.EncodedBlockCert{
		Block: block,
	}

	// Return the same block for round 1-9
	httpmock.RegisterResponder("GET", `=~^http://localhost/v2/blocks/\d?format=msgpack`,
		httpmock.NewStringResponder(200, string(msgpack.Encode(blockCert))))
```